calmTheStorm/calmTheStorm/SurvivalSelection.swift

```diff
@@ -11,25 +11,37 @@ struct SurvivalSelection: View {
     var body: some View {
     ZStack{
         Color.Ivory.ignoresSafeArea(.all)
+        
         VStack{
         Text("Survival Information")
             .font(.custom("Avenir", size: 30))
             .bold()
-            .offset( y:-70)
-        
+            .offset( y:-180)
+            
+            
+            HStack{
+                Image(systemName: "info.circle")
+                    .font(.system(size: 30))
+                    .offset( y:-50)
         Button("Survival Info"){
```

Comment:
Text color is not consistent with home page.  Please fix.

Reply:
@RJB888  Text-color fixed